### PR TITLE
Create SetNewPasswordComponent

### PR DESCRIPTION
**Pull request by tarassito**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LEAR-21)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/registerLogin/SetNewPassword.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    hintText='Enter password'
+                />
+                <br />
+                <h2>Confirm Password</h2>
+                <TextField
+                    onChange={this.handleConfirmPassword}
+                    hintText='Re-enter password'
+                    errorStyle = {errorStyle}
+                    errorText = {this.state.error}
+
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/registerLogin/SetNewPassword.js]: delete empty line

File: eventually/static/src/containers/registerLogin/registrationService.js [hunk 2]
```diff
@@ -23,4 +23,8 @@ const forgetPasswordService = email => {
     return axios.post(forgetPasswordUrl, {email});
 };
 
-export {loginService, forgetPasswordService, registerService, logoutService};
+const putNewPasswordService = (token, new_password) => {
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/registerLogin/registrationService.js]: change new_password to newPassword

File: eventually/static/src/containers/registerLogin/registrationService.js [hunk 3]
```diff
@@ -23,4 +23,8 @@ const forgetPasswordService = email => {
...
+    return axios.put(forgetPasswordUrl+token, {new_password});
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/registerLogin/registrationService.js]: change new_password to newPassword

File: eventually/static/src/containers/registerLogin/SetNewPassword.js (hunk 1 above)
Comment (tarassito) [reply eventually/static/src/containers/registerLogin/SetNewPassword.js]: no problem. Done!

File: eventually/static/src/containers/registerLogin/SetNewPassword.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import {putNewPasswordService} from './registrationService.js';
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/SetNewPassword.js]: remove extensions a file `./registrationService.js` -> `./registrationService`

File: eventually/static/src/containers/registerLogin/SetNewPassword.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+const style = {
+    margin: 11,
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/SetNewPassword.js]: why 11?

Comment (tarassito) [reply eventually/static/src/containers/registerLogin/SetNewPassword.js]: yes, it should be 12px. The same as in Login , Register and ForgetPassword style.

**Review by dotiurin — DISMISSED** [review]

File: eventually/static/src/containers/registerLogin/SetNewPassword.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+    handleSubmit = event => {
+        if (this.state.password == this.state.confirmPassword){
+            const newPassword = this.state.password ;
+            const token = this.state.token;
+            putNewPasswordService(token, newPassword).then((response) => {
+                this.props.history.push('/login');
+            });
+        }
+        else {
+            this.setState({error: 'Passwords does not match'});}
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/SetNewPassword.js]: curly braces move in a new line

**Review by lhalam — CHANGES_REQUESTED** [review]
add validators for fields in UI

**Review by iryn34 — DISMISSED** [review]

**Review by VitalyiKhlivnyi — DISMISSED** [review]

File: eventually/static/src/containers/registerLogin/SetNewPassword.js (hunk 6 above)
Comment (tarassito) [reply eventually/static/src/containers/registerLogin/SetNewPassword.js]: added validators and disable/able button

**Review by lhalam — APPROVED** [review]

**Pull request merged**